### PR TITLE
feat: do not store properties decorated with @Ignore() decorator

### PR DESCRIPTION
**Pull request by danieleisenhardt**
New decorator allows for entities to have properties that are not stored.

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -53,6 +54,13 @@ export function serializeEntity<T extends IEntity>(
   subColMetadata.forEach(scm => {
     delete serializableObj[scm.propertyKey];
   });
+
```
Comment (wovalle) [inline src/utils.ts:57]: Can you create some tests for this case? After serialization, the "ignored" properties shouldn't be serialized

**Review by wovalle — CHANGES_REQUESTED** [review]
Great use of the Reflection api and great work! just a minor comment.

**Comment by wovalle** [comment]
Still missing (will probably create an issue for this):

- One integration test
- Documentation

**Comment by danieleisenhardt** [comment]
> * One integration test
> * Documentation

I'll do my best to add these to the branch.

**Review by wovalle — APPROVED** [review]

**Comment by wovalle** [comment]
Great work! @danieleisenhardt

**Comment by wovalle** [comment]
@all-contributors add @danieleisenhardt for code

**Pull request merged**